client/ayon_core/tools/adobe_webserver/app.py

```diff
@@ -104,14 +104,11 @@ def port_occupied(self, host_name, port):
             again. In that case, use existing running webserver.
             Check here is easier than capturing exception from thread.
         """
-        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        result = True
-        try:
-            sock.bind((host_name, port))
-            result = False
-        except:
-            print("Port is in use")
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as con:
+            result = con.connect_ex((host_name, port)) == 0
 
+        if result:
+            print("Port is in use")
```

Comment:
I wonder if it's worth printing which port is already in use?
```suggestion
        if result:
            print(f"Port {port} is already in use")
```